pkg/controller/template/render.go

```diff
@@ -95,14 +95,7 @@ func GenerateMachineConfigsForRole(config *RenderConfig, role string, path strin
 		return nil, fmt.Errorf("failed to read dir %q: %v", path, err)
 	}
 
-	sshMachineConfigForRole, err := generateSSHConfig(config.SSHKey, role)
```

Comment:
@abhinavdahiya  l ast thing: if we are deleting this, then we dont need the func() generateSSHConfig  (below) anymore right? this is the only spot that calls that function.

Reply:
done

Reply:
thanks!!!